src/index.ts

```diff
@@ -24,7 +25,7 @@ export { config };
 
 export const Context = _context.Context;
 export const useState = _hooks.useState;
-export const core = { EventBus, Observer };
+export const core = { EventBus, Observer, OwlEvent };
```

Comment:
I am not sure that it should be exported...

Reply:
Me neither